app/routes/docs.client.samples.md

```diff
@@ -164,3 +166,62 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## Working With JSON Schema
+
+GCN Notices are distributed in JSON format. This guide describes how to programmatically read JSON schema.
```

Comment:
```suggestion
For new missions, GCN Notices are preferably distributed in JSON format. This guide describes how to programmatically read the JSON schema.
```

Reply:
Why are you making suggestions on your own PR? Just make the change.